drilldown/templates/drilldown/base.html

```diff
@@ -115,6 +118,19 @@
                      $list.querySelectorAll('.add-filter form').forEach((el) => hide(el))
                  }
              }
+
+             function validateZipCode() {
+                let input = $('input[name=zip_code].form-control')
+                input.siblings('.btn').on('click', function(e){
+                    let val = input[0].value
+                    if(isNaN(val) || val.length != 5 || val.charAt(0) != '2'){
+                        e.preventDefault()
+                        input.siblings('.js-error')
+                        .html('NC zip codes have to be 5 digit numeric values starting in \'2\'');
```

Comment:
I'm not seeing these errors when I enter something invalid (abc, 37076, etc). The apply button just doesn't work.